commands/compliment.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+const { avaPrefix } = require('../config/config');
+
+const command = 'compliment';
+const complimentEndpoint = 'https://complimentr.com/api';
+
+module.exports = {
+  commandName: command,
+  name: 'messageCreate',
+  once: false,
+  execute: async(message) => {
+    // Ensure message is intended for ava
+    if (!message.content.startsWith(avaPrefix)){
+      return;
+    }
+    const args = message.content.trim().split(/ +/g);
+    const userCmd = args[1];
+    const userToCompliment = args[2];
+
+    if (userCmd === command) {
+      if (!userToCompliment || !userToCompliment.startsWith('@')) {
```

Comment:
lets just do `!userToCompliment`, some folks may not want to directly tag folks. like `ava compliment me`

Reply:
True, good point.